_GNU_SOURCE is needed for the definition of memrchr. However, since it is
only added to CFLAGS using a conditional assignment operator, and we supply
our own CFLAGS, it is not added to CFLAGS and memrchr is implicitly declared.

This patch fixes this.

diff -upr cpulimit-0.2.orig/src/Makefile cpulimit-0.2/src/Makefile
--- cpulimit-0.2.orig/src/Makefile	2024-05-12 12:59:32.355413480 +0200
+++ cpulimit-0.2/src/Makefile	2024-05-12 13:00:31.805572259 +0200
@@ -1,5 +1,6 @@
 CC?=gcc
-CFLAGS?=-Wall -g -D_GNU_SOURCE
+CFLAGS?=-Wall -g
+CFLAGS+=-D_GNU_SOURCE
 TARGETS=cpulimit
 LIBS=list.o process_iterator.o process_group.o
 
